packages/vue/src/create-element.js

```diff
@@ -1,61 +1,71 @@
+import h from 'hastscript'
+import toH from 'hast-to-hyperscript'
+
 /**
  * MDX default components
  */
-const DEFAULTS = {
+const defaults = {
   inlineCode: 'code',
-  wrapper: 'div'
+  wrapper: {
+    name: 'MDXWrapper',
+    render: function (h) {
+      const children = this.$slots.default
+      return children.length === 1 ? children : h('div', {}, children)
+    }
+  }
 }
 
-/**
- * Renders final tag/component
- * @param {Vue.Component|String} type Element or tag to render
- * @param {Object|Array} props Props and attributes for element
- * @param {Array} children Array of child nodes for component
- * @returns {Vue.VNode} VNode of final rendered element
- */
-export default function(type, props, children) {
-
-  const h = this.createElement
-  const components = this.components
-  const defaults = Object.keys(DEFAULTS)
-
-  let tag
-  let elProps = props
-
-  // We check context to see if the element/tag
-  // is provided in the MDXProvider context.
-  if (Object.keys(components).includes(type)) {
-    // We check to see if props is of type object.
-    // If it is, then we pass them into the MDXContext component
-    const componentProps = typeof props === 'object' ? props : undefined
-    tag = components[type](componentProps)
-
-  } else if (defaults.includes(type)) {
-
-    tag = DEFAULTS[type]
-    // Remove components object from attrs
-    const { components, ...attrs } = elProps.attrs
-    elProps = {
-      attrs
-    }
+const own = {}.hasOwnProperty
+
+export default function (type, props, ...children) {
```

Comment:
for tracing purposes would there be value in giving the function a name?
maybe:
```suggestion
export default function mdxCreateElement(type, props, ...children) {
```
?

Reply:
Good idea! Went w/ `createMdxElement`.

A different issue, but: right now we’re calling everything mdx. Would be good to use `m` / `createMdxElement` for the renderers, and `mdx` for the core which generates serialized JS